src/foundry/client/pixi/layers/effects/illumination-effects.d.mts

```diff
@@ -59,16 +79,49 @@ declare global {
     protected override _draw(options?: Record<string, unknown>): Promise<void>;
 
     protected override _tearDown(options?: Record<string, unknown>): Promise<void>;
+    /**
+     * @deprecated since v11, will be removed in v13
+     * @remarks "CanvasIlluminationEffects#updateGlobalLight has been deprecated."
+     */
+    updateGlobalLight(): false;
 
     /**
-     * Draw illumination baseline
+     * @deprecated since v12, will be removed in v14
      */
-    drawBaseline(): void;
+    background: PIXI.LegacyGraphics;
 
     /**
-     * @deprecated since v11, will be removed in v13
-     * @remarks "CanvasIlluminationEffects#updateGlobalLight has been deprecated."
+     * @deprecated since v12, will be removed in v14
+     * @remarks `"CanvasIlluminationEffects#globalLight has been deprecated without replacement. Check the canvas.environment.globalLightSource.active instead."`
      */
-    updateGlobalLight(): false;
+    get globalLight(): boolean;
+  }
+
+  /**
+   * Cached container used for dynamic darkness level. Display objects (of any type) added to this cached container will
+   * contribute to computing the darkness level of the masked area. Only the red channel is utilized, which corresponds
+   * to the desired darkness level. Other channels are ignored.
+   */
+  class DarknessLevelContainer extends CachedContainer {
+    /**
+     * @defaultValue
+     * ```js
+     * {
+     *  scaleMode: PIXI.SCALE_MODES.NEAREST,
+     *  format: PIXI.FORMATS.RED,
+     *  multisample: PIXI.MSAA_QUALITY.NONE,
+     *  mipmap: PIXI.MIPMAP_MODES.OFF
+     * }
+     * ```
+     */
+    static override textureConfiguration: {
+      multisample?: PIXI.MSAA_QUALITY;
+      scaleMode?: PIXI.SCALE_MODES;
+      format?: PIXI.FORMATS;
+      mipmap?: PIXI.MIPMAP_MODES;
+    };
```

Comment:
Consider the appropriate helper; `InexactPartial`, `NullishProps`, or `IntentionalPartial`.